db/migrate/20221105215547_create_messages.rb

```diff
@@ -0,0 +1,9 @@
+class CreateMessages < ActiveRecord::Migration[7.0]
+  def change
+    create_table :messages do |t|
+      t.text :greetings
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Kindly set up the routes for the new action which returns a random greeting. Also, please remember to add a route for a static page which would be the root of your app. As per [this](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#:~:text=you%20will%20need,of%20your%20app) requirement.